tests/unit_tests/udfs/test_model_downloader_udf.py

```diff
@@ -90,19 +90,18 @@ def test_model_downloader():
             BFS_CONN_NAME)
         result = executor.run([Group([input_data])], exa)
 
-        relative_model_path = str(bucketfs_operations.get_model_path(
-            model_params.sub_dir, model_params.name))
-        full_model_path = pathlib.PurePath(path, relative_model_path)
-        assert result[0].rows[0][0] == relative_model_path \
+        relative_model_path = bucketfs_operations.get_model_path(
+            model_params.sub_dir, model_params.name)
```

Comment:
What happened here?

Reply:
When we use the full_model_path (base_path + relative_path), _get_complete_path_ method in the BucketFSLocation makes full_model_path  relative and then add base_path. So the base bath is dublicated

Reply:
ah ok